sources/packages/backend/apps/workers/src/services/student-assessment/student-assessment.service.ts

```diff
@@ -170,12 +174,81 @@ export class StudentAssessmentService extends RecordDataModelService<StudentAsse
     assessmentId: number,
     status: AssessmentStatus,
   ): Promise<UpdateResult> {
-    return this.repo.update(
+    return await this.dataSource.transaction(
```

Comment:
No need to `await` here, right?

Reply:
mm. Actually... this one I saw this one in a few of our previous PRs. and I wanted to know if sonarcloud will complaint about it or not.

Reply:
Did get the Sonarcloud issue, it was supposed to complain about it unless the rules changed.

Reply:
no... I didn't get any sonar cloud complaints